src/web_performance/content_transformation.conf

```diff
@@ -16,6 +16,11 @@
 # to `off`, also from rewriting other resources.
 #
 # https://developers.google.com/speed/pagespeed/module/configuration#notransform
+#
+# (!) You can test the effects of content transformation applied by
+# Google's Lite Mode by visiting: https://googleweblight.com/i?u=https://www.example.com
+#
+# https://support.google.com/webmasters/answer/6211428?hl=en
```

Comment:
@Malvoz Can you merge this warning with the one above?
Actually a small refactoring would be awesome.

```apacheconf
# (!) Disabling content transformation may also prevent data
#     saver and other content optimization tools.
#     Notably:
#     * `mod_pagespeed`: Header will prevent `PageSpeed` from rewriting
#       HTML files, and, if the `ModPagespeedDisableRewriteOnNoTransform`
#       directive isn't set to `off`, also from rewriting other resources.
#     * Google's Lite Mode: You can test the effects of content
#       transformation applied by visiting:
#       https://googleweblight.com/i?u=https://www.example.com
#
#     https://developers.google.com/speed/pagespeed/module/configuration#notransform
#     https://support.google.com/webmasters/answer/6211428
```

Feel free to review phrasing. 🙂

Reply:
I refactored most of it in https://github.com/h5bp/server-configs-apache/pull/191/commits/e9714082460622507813a047933c3e73a34e651f, WDYT?